src/base/App.js

```diff
@@ -10,9 +10,7 @@ import Routes from './../config/routes';
 
```

Comment:
Since we're cleaning up, can you switch the imports over to absolute paths for `App.js`?

Reply:
Good catch